specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -9134,8 +9272,99 @@
         }
       }
     },
+    "VaultHealthDetails": {
+      "description": "Vault health details definition.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/VaultHealthProperties",
+          "description": "The vault health related data."
+        }
+      }
+    },
+    "VaultHealthProperties": {
+      "description": "class to define the health summary of the Vault.",
+      "type": "object",
+      "properties": {
+        "vaultErrors": {
+          "description": "The list of errors on the vault.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthErrorSummary"
+          }
+        },
+        "protectedItemsHealth": {
+          "$ref": "#/definitions/ResourceHealthSummary",
+          "description": "The list of the health detail of the protected items in the vault."
+        },
+        "fabricsHealth": {
+          "$ref": "#/definitions/ResourceHealthSummary",
+          "description": "The list of the health detail of the fabrics in the vault."
+        }
+      }
+    },
+    "HealthErrorSummary": {
+      "description": "class to define the summary of the health error details.",
+      "type": "object",
+      "properties": {
+        "summaryCode": {
+          "description": "The code of the health error.",
+          "type": "string"
+        },
+        "category": {
+          "description": "The category of the health error.",
+          "type": "string"
+        },
+        "severity": {
+          "description": "Severity of error.",
+          "type": "string"
+        },
+        "summaryMessage": {
+          "description": "The summary message of the health error.",
+          "type": "string"
+        },
+        "affectedResourceType": {
```

Comment:
This is our resource type defined for the proxy resources. This is to identify the resource type of the resources impacted by the certain category of health issue.